CHANGELOG.md

```diff
@@ -16,6 +16,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Disable autocorrect on NoteComposer when running on macOS. [#1460](https://github.com/planetary-social/nos/issues/1460)
 
 ### Internal Changes
+- Moved the database cleanup routine into a background execution task. 
+    [#1426](https://github.com/planetary-social/nos/issues/1426)
```

Comment:
Looks like this somehow got into the wrong section, probably because git diffs are hard.

Reply:
Lately when I merge from `main` there are no conflicts in the changelog, even if there has been a release. Something to be aware of I guess. I'm not sure how to make it better.